wandb/sdk/wandb_config.py

```diff
@@ -113,7 +113,7 @@ def update_locked(self, d, user=None):
     def _sanitize_dict(self, config_dict, allow_val_change=None):
         sanitized = {}
         for k, v in six.iteritems(config_dict):
-            k, v = self._sanitize(k, v)
+            k, v = self._sanitize(k, v, allow_val_change)
```

Comment:
Beast 🐞

Reply:
this was a filed issue, waiting for someone to get to it and add a test.